integration-tests/utils/integration-utils.js

```diff
@@ -106,7 +106,7 @@ const svgText = ({ textContains }) => {
 }
 
 const navChild = text => {
-  return `//*[@role="tablist"]/*[contains(normalize-space(.),"${text}")]`
```

Comment:
I decided to make this test more robust by targeting the nav element by ID instead of by ARIA role. This allows the team more flexibility to refine the roles further in the future